timemachine/md/minimizer.py

```diff
@@ -41,7 +41,7 @@ def bind_potentials(params_potential_pairs):
     return u_impls
 
 
```

Comment:
nit completeness -- also update `get_val_and_grad_fn` in this file?

Reply:
Updated in e3aa8128dc5d17fba300f8c721ef5083f09852b1